google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudExecutionStack.java

```diff
@@ -28,17 +27,19 @@
 import java.util.List;
 
 /**
- * CloudExecutionStack represents an entire stack for a {@link com.google.api.services.debugger.model.BreakPoint} It
- * stores the individual frames, and also the variables and custom watch expressions.
+ * CloudExecutionStack represents an entire stack for a
+ * {@link com.google.api.services.clouddebugger.model.Breakpoint}. It stores the individual frames,
+ * and also the variables and custom watch expressions.
  */
 public class CloudExecutionStack extends XExecutionStack {
   private final List<CloudStackFrame> myFrames = new ArrayList<CloudStackFrame>();
 
   public CloudExecutionStack(@NotNull Project project,
```

Comment:
If you're stacking the params I'd rather have the first one aligned with the rest.